src/test/kotlin/org/rust/ide/structure/RsStructureViewModelTest.kt

```diff
@@ -392,79 +393,92 @@ class RsStructureViewTest : RsTestBase() {
         let var1 = 1;
         let var2: u32 = 2;
         let var3 = "3";
     """, """
-        -null visibility=none
-         var1: i32 visibility=none
-         var2: u32 visibility=none
-         var3: &str visibility=none
+        |-null visibility=none
+        | var1: i32 visibility=none
+        | var2: u32 visibility=none
+        | var3: &str visibility=none
     """)
 
     fun `test console variables destructuring tuples and arrays`() = doTestForREPL("""
         let (var1, (var2, var3)) = (1i32, (2u32, "3"));
         let [var4, var5] = [1, 2];
     """, """
-        -null visibility=none
-         var1: i32 visibility=none
-         var2: u32 visibility=none
-         var3: &str visibility=none
-         var4: i32 visibility=none
-         var5: i32 visibility=none
+        |-null visibility=none
+        | var1: i32 visibility=none
+        | var2: u32 visibility=none
+        | var3: &str visibility=none
+        | var4: i32 visibility=none
+        | var5: i32 visibility=none
     """)
 
     fun `test console variables destructuring struct`() = doTestForREPL("""
         struct Struct1 { foo: u32 }
         struct Struct2 { field1: u16, field2: Struct1 }
         let struct2 = Struct2 { field1: 1, field2: Struct1 { foo: 0 } };
         let Struct2 { field1: var1, field2: Struct1 { foo } } = struct2;
-    """, """
-        -null visibility=none
-         -Struct1 visibility=private
-          foo: u32 visibility=private
-         -Struct2 visibility=private
-          field1: u16 visibility=private
-          field2: Struct1 visibility=private
-         struct2: Struct2 visibility=none
-         var1: u16 visibility=none
-         foo: u32 visibility=none
+    """,
+        """
+        |-null visibility=none
+        | -Struct1 visibility=private
+        |  foo: u32 visibility=private
+        | -Struct2 visibility=private
+        |  field1: u16 visibility=private
+        |  field2: Struct1 visibility=private
+        | struct2: Struct2 visibility=none
+        | var1: u16 visibility=none
+        | foo: u32 visibility=none
     """)
 
     fun `test console variables destructuring enum`() = doTestForREPL("""
         enum E { Variant(u8) }
         let var1 @ E::Variant(..) = E::Variant(1);
     """, """
-        -null visibility=none
-         -E visibility=private
-          Variant(u8) visibility=none
-         var1: E visibility=none
+        |-null visibility=none
+        | -E visibility=private
+        |  Variant(u8) visibility=none
+        | var1: E visibility=none
     """)
 
     private fun doTest(@Language("Rust") code: String, expected: String, fileName: String = "main.rs") {
-        val normExpected = expected.trimIndent()
+        doTestWithActions(code, expected, fileName) {}
+    }
+
+    private fun doTestForREPL(code: String, expected: String) = doTest(code, expected, RsConsoleView.VIRTUAL_FILE_NAME)
+}
+
+abstract class RsStructureViewTest : RsTestBase() {
```

Comment:
Could you please rename it to `RsStructureViewTestBase` to be uniform with other "base" test classes (like `RsTestBase`). Let's also extract the class to a separate file